pipelines/fbc-release/README.md

```diff
@@ -17,6 +17,10 @@ Tekton release pipeline to interact with FBC Pipeline
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                    | Yes       | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                           | Yes       | main                                                            |
 
+### Changes in 1.6.0
+- modify the task `publish-index-image` to accept the new parameter `buildTimestamp` used
```

Comment:
This wording would be appropriate for the task update, but it doesn't read right here. It should say that we added the new parameter when calling the task. Something like:
```
- add new parameter `buildTimestamp` when calling the `publish-index-image` task
  * this new parameter is used for adding a timestamped tag to the index image
```